src/components/widgets/AssocListPopup/AssocListPopup.tsx

```diff
@@ -102,10 +179,17 @@ export const AssocListPopup: FunctionComponent<IAssocListProps & IAssocListActio
 function mapStateToProps(store: Store, ownProps: IAssocListOwnProps) {
     const bcName = ownProps.widget.bcName
     const bc = store.screen.bo.bc[bcName]
+    const pendingDataChanges = store.view.pendingDataChanges
+    const isFilter = store.view.popupData.isFilter
+    const calleeBCName = store.view.popupData.calleeBCName
     return {
         showed: store.view.popupData.bcName === ownProps.widget.bcName,
         assocValueKey: store.view.popupData.assocValueKey,
-        bcLoading: bc?.loading
+        associateFieldKey: store.view.popupData.associateFieldKey,
+        bcLoading: bc?.loading,
+        pendingDataChanges: pendingDataChanges,
```

Comment:
It's not clear why you need a separate variable.
```tsx
const pendingDataChanges = store.view.pendingDataChanges
return {
    // ...
    pendingDataChanges
}
```
or
```tsx
return {
    // ...
    pendingDataChanges: store.view.pendingDataChanges
}

Reply:
fixed